test/sharness/t0100-cleanup.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# MIT Licensed
+#
+
+test_description="Cleanup state from tests (kill ipfs daemon)"
```

Comment:
cleanups should happen within each sharness test file. This means initialize/kill ipfs and cluster on each test file. Every test file should be self-contained: can be run separately and when it's done there is no trash lying around.